### PR TITLE
ddl: Fix issue 2293

### PR DESCRIPTION
**Pull request by zimulala**
Fix issue #2293

### CODE REVIEW

**Comment by hanfei1991** [comment]
LGTM

**Comment by zimulala** [comment]
PTAL @tiancaiamao @coocood

File: ddl/ddl.go [hunk 1]
```diff
@@ -1022,6 +1022,15 @@ func (d *ddl) AddColumn(ctx context.Context, ti ast.Ident, spec *ast.AlterTableS
 		return errors.Trace(err)
 	}
 
+	// Check column default value.
```
Comment (tiancaiamao) [inline ddl/ddl.go]: line 1018, typo

> later we will change the column's offset later.

**Comment by tiancaiamao** [comment]
LGTM

File: ddl/ddl.go (hunk 1 above)
Comment (zimulala) [inline ddl/ddl.go]: I will fix it in next PR.

**Pull request merged**